_releases/2025-04-09-1.16.0-released.md

```diff
@@ -25,6 +25,8 @@ For more details, visit the [full changelog](https://github.com/crystal-lang/cry
 
 ### Breaking
 
+The following changes break prior behavior of the compiler, but we expect them to not break much in existing code. If this is not the case, please let us know in the [issue tracker] or [forum].
```

Comment:
Sorry, this wasn't a complete suggestion; it's missing the references

Reply:
That's the problem with accepting suggestions directly in GitHub's UI. It doesn't go through pre-commit and I'm used to it double-checking my changes 😅